### DISCUSSION TITLE
Change title

### DISCUSSION THREAD

**Discussion by bubfcg** [code4recovery/12-step-meeting-list#1426] [Q&A]
Is there any way I can change the `<h1>` title of the meeting list page and also the `<title>` tag?

**Comment by joshreisner** [comment]
<img width="1265" alt="Screenshot 2024-05-22 at 7 19 46 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/9a0c5806-2eb1-4265-b437-8c8bdb19e8fc">

the text that reads "Meetings in Campbell" in this screenshot? what did you want it to say?
